lib/classes/Swift/Transport/MailTransport.php

```diff
@@ -249,7 +278,9 @@ private function _getReversePath(Swift_Mime_Message $message)
     private function _formatExtraParams($extraParams, $reversePath)
     {
         if (false !== strpos($extraParams, '-f%s')) {
-            // no need to escape $reversePath) as mail() already does it
+            if (false === $this->_isShellSafe($reversePath)) {
```

Comment:
Unlike on line 200, there's no need to explicitly check for `false` here.  `if (!$this->_isShellSafe($reversePath)` should be sufficient.